File: 프로그래머스/0/181847. 0 떼기/0 떼기.js
function solution(n_str) {
    var answer = '';
    
    var arr = n_str.split("");
    
    for(var i = 0; i < arr.length; i++) {
        if(arr[i] === '0') {
            continue;
        } else {
            answer = n_str.substr(n_str.indexOf(arr[i]));
            break;
        }
    }
    
    return answer;
}

File: 프로그래머스/0/181847. 0 떼기/0 떼기.java
class Solution {
    public String solution(String n_str) {
        StringBuilder answer = new StringBuilder();
        String[] strArray = n_str.split("");
        
        for(int i = 0; i < strArray.length; i++) {
            if(strArray[i].equals("0")) {
                continue;
            } else {
                answer.append(n_str.substring(i));
                break;
            }
        }
        
        return answer.toString();
    }
}